Readme.md

```diff
@@ -9,10 +9,10 @@ Naggum does not aim to be yet another Common Lisp or Scheme or Emacs Lisp or
 whatever implementation. Instead, we are trying to deliver a modern Lisp dialect
 that makes use of most of .Net's benefits.
 
-## Build
+## Documentation
 
-To build the project under various environments, see [build
-instructions][build-instructions].
+To read the build or usage documentation and get familiar with Naggum syntax
+visit [the documentation site][read-the-docs].
```

Comment:
IMO, the sentence is overly long. "The documentation is available on [ReadTheDocs][read-the-docs]" should do just fine.